composer.json

```diff
@@ -44,7 +44,8 @@
         "dealerdirect/phpcodesniffer-composer-installer": "*",
         "mglaman/phpstan-drupal": "^1.1",
         "phpstan/extension-installer": "^1.1",
-        "phpstan/phpstan-deprecation-rules": "^1.0"
+        "phpstan/phpstan-deprecation-rules": "^1.0",
+        "webflo/drupal-finder": "^1.3"
```

Comment:
I'm also adding requirement to version as the new class was added in 1.3 https://github.com/webflo/drupal-finder/compare/1.2.2...1.3.0#diff-19dafcffb2094365c57f581c138aede2762e021675c6d8dab34c5dbb78508128R1